resources/js/i18n/messages/ar/comps.js

```diff
@@ -1,6 +1,8 @@
 export default {
-  // example: {
```

Comment:
please keep this example. and move the new translations to a new file called `msgs.js`